Update ClickHouse sink (#2682)
risingwavelabs/risingwave-docs: docs/guides/sink-to-clickhouse.md

| `clickhouse.database`  | Required. Name of the ClickHouse database that you want to sink data to.|
| `clickhouse.table`      | Required. Name of the ClickHouse table that you want to sink data to.|
| `commit_checkpoint_interval`| Optional. Commit every N checkpoints (N > 0). Default value is 10. <br/>The behavior of this field also depends on the `sink_decouple` setting:<ul><li>If `sink_decouple` is true (the default), the default value of `commit_checkpoint_interval` is 10.</li> <li>If `sink_decouple` is set to false, the default value of `commit_checkpoint_interval` is 1.</li> <li>If `sink_decouple` is set to false and `commit_checkpoint_interval` is set to larger than 1, an error will occur.</li></ul>|
| `clickhouse.delete.column` | Optional. Add this parameter when using ClickHouse's ReplacingMergeTree and setting up the delete column. You can run an upsert sink using the ReplacingMergeTree engine.|

### Upsert sinks

While RisingWave supports `append-only` sinks for all ClickHouse engines, support for `upsert` sinks is limited. Additionally, for ReplacingMergeTree engines, an `append-only` sink will not insert duplicate data.

> We support creating `upsert` sinks for CollapsingMergeTree and VersionedCollapsingMergeTree engines. RisingWave will transform `DELETE` into `INSERT SIGN = -1`.

Author: Yes this looks more organized. Updated.

| `clickhouse.database`  | Required. Name of the ClickHouse database that you want to sink data to.|
| `clickhouse.table`      | Required. Name of the ClickHouse table that you want to sink data to.|
| `commit_checkpoint_interval`| Optional. Commit every N checkpoints (N > 0). Default value is 10. <br/>The behavior of this field also depends on the `sink_decouple` setting:<ul><li>If `sink_decouple` is true (the default), the default value of `commit_checkpoint_interval` is 10.</li> <li>If `sink_decouple` is set to false, the default value of `commit_checkpoint_interval` is 1.</li> <li>If `sink_decouple` is set to false and `commit_checkpoint_interval` is set to larger than 1, an error will occur.</li></ul>|

> | `clickhouse.delete.column` | Optional. Add this parameter when using ClickHouse's ReplacingMergeTree and setting up the delete column. You can run an upsert sink using the ReplacingMergeTree engine.|

Reviewer: ```suggestion
| `clickhouse.delete.column` | Optional. You can run an upsert sink using the ReplacingMergeTree engine. When using the ReplacingMergeTree engine, you can specify the delete column with this parameter. |
```

Reviewer: Not sure if my rewording is accurate. @xxhZs Please review. Thanks.

### Upsert sinks

While RisingWave supports `append-only` sinks for all ClickHouse engines, support for `upsert` sinks is limited. Additionally, for ReplacingMergeTree engines, an `append-only` sink will not insert duplicate data.

> RisingWave supports `upsert` sinks for the following ClickHouse engines:

Reviewer: Better than my version. Awesome! Thank you.
